ansible/playbook/kubernetes/README.adoc

```diff
@@ -92,6 +92,75 @@ ansible-playbook ansible/playbook/kubernetes/k8s_remove.yml -e vm_name=${VM_NAME
 
 Several tools can be installed using the generic install tooks playbook. To deploy other k8s tools.
 
+.Common parameters
+[cols="2,5"]
+|===
+| Parameter | Description
+
+| `vm_name`
+
+[.fuchsia]#string#
+
+[.red]#required# 
+
+a| Name of the k8s host to apply the configuration
+
+| `state`
+
+[.fuchsia]#string#
+
+[.red]#required# 
+
+a| One of:
+
+* `present`: Install
+* `absent`: Uninstall
+
+|===
+
+
+=== k8s_cert_manager
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/kubernetes/k8s_install_tools.yml \
+  -e vm_name=${VM_NAME} -e state=present \
+  --tags k8s_cert_manager
+----
+
+As a result the following resources should be available.
+
+[source]
+----
+$ kubectl -n cert-manager get all
+NAME                                           READY   STATUS    RESTARTS   AGE
+pod/cert-manager-7c869867bf-jlwxj              1/1     Running   0          88s
+pod/cert-manager-cainjector-54c9d9b775-9zhrm   1/1     Running   0          88s
+pod/cert-manager-webhook-7f7469bdb7-m5b5z      1/1     Running   0          88s
+pod/godaddy-webhook-c6b5f74fd-5w7q6            1/1     Running   0          66s
```

Comment:
Can you also, part of the description, mention that we install also the Godaddy webhook - https://github.com/snowdrop/k8s-infra/blob/main/ansible/roles/cert_manager/tasks/install.yml#L24 ?

Reply:
Added to the docs.